### PR TITLE
Simplify and refactor parallax barrier example

### PR DESCRIPTION
**Pull request by looeee**
Replaced all the car models with a single car and removed options to change car colours, added a ground and removed use of BinaryLoader (model is now FBX). 

[Live version](https://rawgit.com/looeee/three.js/d78c734dd2dbe565eed5cde17bd83fe78629a571/examples/webgl_effects_parallaxbarrier.html)

### CODE REVIEW

**Comment by moraxy** [comment]
On a proper GPU it's fine, but on my notebook's integrated Intel HD I'm getting just a bit over 10% of the framerate compared to [the example on threejs.org](https://threejs.org/examples/webgl_effects_parallaxbarrier.html) (~55 vs. ~6 FPS).  
Is the model particularly detailed or so?

**Comment by looeee** [comment]
No, it shouldn't be - its a little over 130k polys which is roughly the same as the current models, and the file size is about 5mb compared to about 3.5mb for the binary format versions. 

I just tested on my phone and performance is pretty terrible there tii though. I'l try changing the format and see if that helps.

**Comment by looeee** [comment]
I've switched to a GLTF version of the model and I'm now getting 15-20fps on my mobile. Still surprisingly slow. 

New live version](https://rawgit.com/looeee/three.js/589a8553cc945cf0537b47005513ceca1b836c0b/examples/webgl_effects_parallaxbarrier.html)

**Comment by moraxy** [comment]
I'm getting about 25fps now, so definitely an improvement here as well.  
Also, resizing the window now increases fps up to 30+, that didn't have any effect on the previous version.

**Comment by mrdoob** [comment]
Hmm, I'll refactor the cars example tomorrow. Try to use a model that it's on the repo already for now.

**Comment by looeee** [comment]
I've converted the F50 model from the original example to GLTF format. 

[New live version](https://rawgit.com/looeee/three.js/b7e4d7b430540e2afe372b1a29d662af2ceaf262/examples/webgl_effects_parallaxbarrier.html)

@mrdoob shall I convert the other car models to GLTF as well?

**Comment by looeee** [comment]
As suggested by @Mugen87 in #13651, changed example to be equivalent to the anaglyph example so that they can be easily compared. 
[New live version](https://rawgit.com/looeee/three.js/f000465b3d1ee9264cd5c520123d0d41e6adb4a5/examples/webgl_effects_parallaxbarrier.html)

File: examples/webgl_effects_parallaxbarrier.html [hunk 1]
```diff
@@ -1,157 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
-		<title>three.js webgl - effects - parallax barrier</title>
+		<title>three.js webgl - effects - anaglyph</title>
```
Comment (WestLangley) [inline examples/webgl_effects_parallaxbarrier.html]: oops.

Comment (looeee) [reply examples/webgl_effects_parallaxbarrier.html]: 😅

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
@mrdoob Can we merge this one? In this way, we can make a little bit progress in removing `THREE.BinaryLoader` 😇.

**Comment by looeee** [comment]
/ping @mrdoob are you likely to merge this? I'll close it if not.

**Comment by Mugen87** [comment]
Latest live link: https://rawgit.com/looeee/three.js/refactor_parallax_example/examples/webgl_effects_parallaxbarrier.html

I vote for a merge 😊

**Comment by mrdoob** [comment]
Sorry about the delay.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**